### ISSUE TITLE
Aligning signup form fields with Mobile Team

### ISSUE THREAD

**Issue by burakmert236** [bounswe/bounswe2022group5#214]
***Description*:**

Aligning all signup form fields with mobile team. 

***Todo's*:**
- [x] Screen shots of frontend signup pages will be provided.

***Reviewer*:** @kardelendemiral @palahb 

***Task Deadline*:** 29/10/2022 23:00
***Review Deadline*:** 30/10/2022 12:00

**Comment by burakmert236** [comment]
User signup form fields: 

<img width="928" alt="Ekran Resmi 2022-10-25 19 21 21" src="https://user-images.githubusercontent.com/73586955/197830431-29d7093c-2356-4578-9165-200b86fee8f8.png">

Doctor signup form fields:  

<img width="907" alt="Ekran Resmi 2022-10-25 19 21 27" src="https://user-images.githubusercontent.com/73586955/197830552-1045a004-6532-431b-b990-6d0e8ca3e736.png">

**Comment by palahb** [comment]
@kardelendemiral's implementation for mobile sign-up screen is as follows:

<img width="192" alt="Ekran Resmi 2022-10-25 19 47 54" src="https://user-images.githubusercontent.com/74200767/197836688-b4e64aa9-2481-491e-827a-e7b284fd58ba.png"><img width="195" alt="Ekran Resmi 2022-10-25 19 48 05" src="https://user-images.githubusercontent.com/74200767/197836707-0e8f4220-d1b1-41ac-88de-ef20d0037ffd.png">

For sign-up screens, [Sign-up requirements](https://github.com/bounswe/bounswe2022group5/wiki/Requirements#1111-sign-up) should be taken into consideration.

There are some differences between these implementations:
1. In terms of user type names, we decided that there are two types of users: **Member**s and **Doctor**s. In frontend, instead of _User membership_, the terminology we decided can better be used I think. These changes can be done in the frontend:
    - **User Membership** => **Sign up as Member**
    - **Doctor Membership** => **Sign up as Doctor**
2. We decided that member sign up does not require name and surname submission. As we stated here,  _**Personal Information**_ includes _a registered user's full-name, location, medical history, weight, height, and age information_. These can be submitted by a user after signing up. I think name and last name can be ignored for member sign up.
3. According to requirement **1.1.1.1.9**, all registered users _**shall**_ provide their date of births. What I get from this requirement is that it is mandatory for a user to provide their date of bitrths. In frontend it seems date of birth is optional. I think it should be mandatory. 
4. As we have decided before, there will not be phone number verification. Also, there is no requirement about providing phone numbers for any user type. In frontend implementation for doctor sign-up, it seems like a mandatory phone number registration is present. I think this should be removed. 
5. According to requirement **1.1.1.1.14**, doctors should provide categories that they are specialized. It seems the _**branch**_ field in frontend serves this purpose, but there is no field for this in mobile screen. It can be added.
6. I could not get what _**title**_ is in the frontend doctor signup screen, I could not find a requirement or word in glossary that describes this or requiring submitting this info while registering. Maybe it can be discarded.

@burakmert236 and @kardelendemiral , what do you think about these points?
These are my thoughts about some differences. I may have missed some points. Please correct me if so.

**Comment by burakmert236** [comment]
I changed title of the forms to replace user with member.
I removed first name and last name fields for members.
I made date of birth field mandatory.
I removed phone number field form doctors' form.
I removed title field from doctors' form

New version of doctors' form:

<img width="500" alt="Ekran Resmi 2022-10-26 20 46 45" src="https://user-images.githubusercontent.com/73586955/198099964-5d342b40-c481-4bb5-8085-195fe944f9e4.png">

New version of members' form:

<img width="500" alt="Ekran Resmi 2022-10-26 20 50 59" src="https://user-images.githubusercontent.com/73586955/198100053-0b8cb329-0ede-4fb1-ab22-89e0d3dd2bb4.png">
